src/shared/Logo/ui/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { LOGO_URL } from "../lib/constants"
+
+export interface LogoProps {
+  widthPx?: number
+  heightPx?: number
+  classNameExtended?: string
```

Comment:
No need to put the word "Extra" here. You can go with just `className` and it'll match with the React props style

Reply:
Done